public_html/register.php

```diff
@@ -1,18 +1,28 @@
 <?php
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
```

Comment:
You don't want to add these variables here, instead look at [README](https://github.com/VictorWinberg/eitf05/blob/master/README.md#amp---display-errors) file.